front/components/members/MembersList.tsx

```diff
@@ -149,7 +149,7 @@ export function MembersList({
       <ChangeMemberModal
         onClose={() => setSelectedMember(null)}
         member={selectedMember}
-        owner={owner}
+        mutateMembers={mutateMembers}
```

Comment:
Passing it directly to the child component is simpler than calling a new useSearchMembers with disabled = true on the child component, since with this option we don't need to pass (searchTerm/pagination data).

Reply:
Nice! 🙏🏻